data/processed/hesa/fte_research_students.yaml

```diff
@@ -0,0 +1,22 @@
+api: the type of API to be used to fetch data: Download
+api_documentation: https://www.hesa.ac.uk/data-and-analysis/estates/table-1
+description: Data about university estates and financials
+endpoint_source: https://www.hesa.ac.uk/data-and-analysis/estates/data.csv
+framework_group: 1
+is_experimental: False
+nuts_level: 2
+source: the data provider: HESA (Higher Education Statistical Agency) https://www.hesa.ac.uk/
+year_range: ['2015/2016', '2017/18']
+schema:
+  nuts_id:
+    type: NutsRegion.id
+  nuts_year_spec:
+    type: NutsRegion.year_spec
+  value:
+    description: Full Time Equivalent (FTE) research students
+    id: fte_research_students
+    label: Research students (Full Time Equivalent)
+    type: int
```

Comment:
Why are we using `int` here instead of FTEs?

Reply:
The values in the dataset are floats, what about this?

Reply:
We don't have any floats in this table.